samples/ingestion/ingestion-client-portal/IngestionClientPortal/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "www": "^0.0.0"
+  },
+  "devDependencies": {
+    "@angular/cli": "^13.3.3"
```

Comment:
Here is still the trace of angular. Might need to do a npm uninstall

Reply:
removed the dependency now, and also removed the package-lock.json